lock-api/src/main/java/com/palantir/lock/client/DialogueAdaptingConjureTimelockService.java

```diff
@@ -85,4 +90,10 @@ public GetCommitTimestampsResponse getCommitTimestamps(AuthHeader authHeader, St
             GetCommitTimestampsRequest request) {
         return dialogueDelegate.getCommitTimestamps(authHeader, namespace, request);
     }
+
+    private <T> T executeInTimerContext(Supplier<T> supplier) {
+        try (Timer.Context timer = this.conjureTimelockServiceBlockingMetrics.startTransactions().time()) {
```

Comment:
nit: We don't normally use `this` when referring to fields (outside of assigning them in a constructor, or if they are needed e.g. when referencing things from a parent class outside of an inner class).